src/openrct2/cheats.c

```diff
@@ -420,6 +422,58 @@ static void cheat_set_staff_speed(uint8 value)
 	}
 }
 
+static void cheat_own_all_land() {
+	uint8 prev_sandbox, prev_no_money, i;
+	uint16 spriteIndex, nextSpriteIndex, spawns_x[2];
+	int min, max, flags, cmd, type;
+	rct_peep *peep;
+
+	//Store states of sandbox, no money, and spawn points to restore when finished
+	prev_sandbox = gCheatsSandboxMode;
+	prev_no_money = gParkFlags & PARK_FLAGS_NO_MONEY;
+	for (i = 0; i < 2; i++) {
+		spawns_x[i] = gPeepSpawns[i].x;
+	}
+
+	//Sandbox mode is needed for this. Also a cheat should not cost, so enable no money.
+	gCheatsSandboxMode = true;
+	if (!prev_no_money)
+		gParkFlags |= PARK_FLAGS_NO_MONEY;
+
+	//Own the entire map
+	min = 32;
+	max = gMapSizeUnits - 32;
+	type = 2;
+	flags = GAME_COMMAND_FLAG_APPLY;
+	cmd = GAME_COMMAND_SET_LAND_OWNERSHIP;
+	game_command_set_land_ownership(&min, &flags, &min, &type, &cmd, &max, &max);
+
+	//remove path from map edge to park entrances
+	editor_check_park_entrance_paths();
+
+	//Go through and remove any guests that get stuck inside the park
+	for (spriteIndex = gSpriteListHead[SPRITE_LIST_PEEP]; spriteIndex != SPRITE_INDEX_NULL; spriteIndex = nextSpriteIndex) {
+		peep = &(get_sprite(spriteIndex)->peep);
+		nextSpriteIndex = peep->next;
+		if (peep->type == PEEP_TYPE_GUEST) {
+			if (peep->outside_of_park) {
+				if (map_is_location_in_park(peep->x, peep->y)) {
+					peep_remove(peep);
```

Comment:
Nah, they'd still be stuck. I found just enough to mark the guest as inside the park, but it feels very hackish and i fear may risk breaking the game, especially with having to modify gNumGuestsHeadingForPark without necessarily knowing whether or not they were heading for the park or if they were leaving (when a guest leaves the park, gNumGuestsHeadingForPark doesn't get incremented).

This is the code that, if replacing `peep_remove(peep);` with, it will mark them as inside the park and can be dragged.
```
peep->outside_of_park = 0;
peep->time_in_park = gScenarioTicks;
peep->state = PEEP_STATE_WALKING;
gNumGuestsInPark++;
gNumGuestsHeadingForPark--;
```

Another thing to think of, even if they were marked safely as in the park, situations such as http://i.imgur.com/VkBspra.png , would leave a guest stuck on a path. if a player doesn't see that right away, the guest will wonder back and forth and will lower the park rating. I just felt overall it was best to keep it simple and remove them from existence. 

However, that being said, i could check into seeing if i can simply move them in front of the park entrance. Would that be good enough? that shouldn't require much ai modification and thus not risk breaking the game.